# HG changeset patch
# User Sam Lantinga <slouken@libsdl.org>
# Date 1362007231 28800
# Branch SDL-1.2
# Node ID b6b2829cd7ef743c6421fdcf5baf21200e4bf474
# Parent  6cccc0beb880694ef84c8333ff734c6549c3ff21
Allow "pulseaudio" as well as "pulse" so SDL 1.2 and 2.0 support both names for the pulseaudio driver.

diff -r 6cccc0beb880 -r b6b2829cd7ef src/audio/SDL_audio.c
--- a/src/audio/SDL_audio.c	Tue Feb 19 22:05:52 2013 +0000
+++ b/src/audio/SDL_audio.c	Wed Feb 27 15:20:31 2013 -0800
@@ -310,6 +310,11 @@
 		SDL_AudioQuit();
 	}
 
+	/* SDL 2.0 uses the name "pulseaudio", so we'll support both */
+	if ( driver_name && SDL_strcasecmp(driver_name, "pulseaudio") == 0 ) {
+		driver_name = "pulse";
+	}
+
 	/* Select the proper audio driver */
 	audio = NULL;
 	idx = 0;
